### PR TITLE
Using EmailJS to notify user when profile changes

### PR DESCRIPTION
**Pull request by Eeyore6**
The feature could be used when cards/ boards .. change.

image
Red lines could be replace with other words

### CODE REVIEW

File: frontend/src/components/common/SendEmailNotification/SendEmailNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        to_email,
+        to_name,
+        subject,
+        message,
+    };
+
+    import('emailjs-com')
+        .then(emailjs => {
+            emailjs
+                .send('service_rkmqc9w', 'template_h8ylqcg', templateParams, 'yo_kKRN2eG451iE_u')
```
Comment (mursal-ashraf) [inline frontend/src/components/common/SendEmailNotification/SendEmailNotification.tsx]: Can you add the service ID as an environment variable in the `.env` file?

**Review by mursal-ashraf — APPROVED** [review]
Add a screenshot of one example email sent, and 1 minor change. Then good to merge. Nice work!

File: frontend/src/components/common/SendEmailNotification/SendEmailNotification.tsx (hunk 1 above)
Comment (Eeyore6) [reply frontend/src/components/common/SendEmailNotification/SendEmailNotification.tsx]: Here is the sample email screenshot:
image

Comment (Eeyore6) [reply frontend/src/components/common/SendEmailNotification/SendEmailNotification.tsx]: I created an account named: kanban.official@gmail.com, the password is kanbanboard. Seems like a professional team XD

**Pull request merged**